src/cascadia/TerminalSettingsEditor/Resources/de-DE/Resources.resw

```diff
@@ -311,13 +311,13 @@
     <value>Die Terminalanwendung, die gestartet wird, wenn eine Befehlszeilenanwendung ohne vorhandene Sitzung gestartet wird, beispielsweise vom Startmenü oder über das Dialogfeld "Ausführen".</value>
     <comment>A description to clarify that the dropdown choice for default terminal will tell the operating system which Terminal application (Windows Terminal, the preview build, the legacy inbox window, or a 3rd party one) to use when starting a command line tool like CMD or Powershell that does not already have a window.</comment>
   </data>
-  <data name="Globals_ForceFullRepaint.Header" xml:space="preserve">
+  <data name="Globals_DisablePartialInvalidation.Header" xml:space="preserve">

```

Comment:
unfortunately, this is well-meaning but will not do much. localization will see Globals_DisablePartialInvalidation as a completely new key, and relocalize it. it'll overwrite these either way :)

Reply:
you could reduce the loc burden by keeping the Uid the same (alas)

Reply:
> you could reduce the loc burden by keeping the Uid the same (alas)

Since I rewrote the text I felt like this will be fine. After all, we'll be getting a new translation anyway.

Reply:
fine with me!